packages/ui/atomic/health-check/src/schema.ts

```diff
@@ -0,0 +1,48 @@
+import {z} from 'zod';
+
+export default z.object({
```

Comment:
@jpmarceau , I would need your stamp on the wording. 🙏
You can refer to the PR images for more info on how it will be displayed.